research/src/pages/prototypes/selectmenu.mdx

```diff
@@ -7,7 +7,13 @@ showInMenu: false
 import SelectAnatomy from '../../components/select-anatomy'
 import Image from '../../components/image'
 
-# How to use the &lt;selectmenu&gt; component
+# &lt;selectmenu&gt; element explainer
+
+## Background
+
+The `<select>` element does not provide enough customization for web developers, which leads them to implement their own and likely miss out on the performance, reliability, and accessibility of built-in form control elements. More on that is in the [select proposal](../components/select) and [Custom Control UI](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ControlUICustomization/explainer.md#introduction).
```

Comment:
"which leads to them _implementing_ their"

Reply:
Can we break up the two sentences?

The `<select>` element does not provide enough customization for web developers, which leads them to implementing their own. These implementations can lead to reduced performance, reliability, and accessibility over the native form control elements.

Reply:
done